jina/serve/networking.py

```diff
@@ -326,18 +353,40 @@ async def send_requests(
                         compression=compression,
                         timeout=timeout,
                     )
-                    with self._summary_time:
+                    if self._metrics.send_requests_bytes_metrics:
+                        self._metrics.send_requests_bytes_metrics.observe(
+                            requests[0].nbytes
```

Comment:
why just `requests[0]`

Reply:
here len(requests) == 1 as you can see in the above line